packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -307,21 +307,29 @@ describe('Combobox', () => {
             {
                 name: ComboboxAutocomplete.inline
             },
+            {
+                name: ComboboxAutocomplete.list
+            },
             {
                 name: ComboboxAutocomplete.both
             }
         ] as const;
-        parameterizeSpec(filterOptionTestData, (spec, name) => {
-            spec(
-                `disabled options will not be selected by keyboard input with autocomplete "${name}"`,
-                async () => {
+        parameterizeSuite(filterOptionTestData, (suite, name) => {
+            suite(`with autocomplete "${name}"`, () => {
+                it('will not autocomplete disabled option', async () => {
                     element.autocomplete = name;
                     pageObject.setInputText('F');
                     await pageObject.clickAway(); // attempt to commit typed value
 
                     expect(element.value).not.toEqual('Four');
-                }
-            );
+                });
+
+                it('allows setting disabled option if typed in', () => {
```

Comment:
This name seems a little misleading as we aren't allowing the _setting_ of the value to the disabled option, just that after completing typing of the same text as a disabled option, we don't immediately switch it to something else. Perhaps "will not switch option if disabled option typed in fully"?

Reply:
> we aren't allowing the setting of the value to the disabled option

But typing in the option _does_ set the control's value. Am I misunderstanding you?

Reply:
Changed the tests to do the same as the existing test (commit by clicking away).